aspnet/mvc/overview/older-versions-1/nerddinner/build-a-model-with-business-rule-validations.md

```diff
@@ -43,41 +43,41 @@ We can use the LINQ language support within VB and C# to write expressive querie
 
-![](build-a-model-with-business-rule-validations/_static/image4.png)
+![Screenshot of Server Explorer. Tables is expanded. Dinners and R S V P are highlighted.](build-a-model-with-business-rule-validations/_static/image4.png)
 
 We can then drag the tables onto the LINQ to SQL designer surface. When we do this LINQ to SQL will automatically create Dinner and RSVP classes using the schema of the tables (with class properties that map to the database table columns):
 
-![](build-a-model-with-business-rule-validations/_static/image5.png)
+![Screenshot of the Nerd Dinner dialog box. The Dinner and R S V P classes are shown.](build-a-model-with-business-rule-validations/_static/image5.png)
 
 By default the LINQ to SQL designer automatically "pluralizes" table and column names when it creates classes based on a database schema. For example: the "Dinners" table in our example above resulted in a "Dinner" class. This class naming helps make our models consistent with .NET naming conventions, and I usually find that having the designer fix this up convenient (especially when adding lots of tables). If you don't like the name of a class or property that the designer generates, though, you can always override it and change it to any name you want. You can do this either by editing the entity/property name in-line within the designer or by modifying it via the property grid.
 
 By default the LINQ to SQL designer also inspects the primary key/foreign key relationships of the tables, and based on them automatically creates default "relationship associations" between the different model classes it creates. For example, when we dragged the Dinners and RSVP tables onto the LINQ to SQL designer a one-to-many relationship association between the two was inferred based on the fact that the RSVP table had a foreign-key to the Dinners table (this is indicated by the arrow in the designer):
 
-![](build-a-model-with-business-rule-validations/_static/image6.png)
+![Screenshot of the Dinner and R S V P tables. An arrow is highlighted and pointing from the Dinner properties tree and R S V P properties tree.](build-a-model-with-business-rule-validations/_static/image6.png)
 
 The above association will cause LINQ to SQL to add a strongly typed "Dinner" property to the RSVP class that developers can use to access the Dinner associated with a given RSVP. It will also cause the Dinner class to have a "RSVPs" collection property that enables developers to retrieve and update RSVP objects associated with a particular Dinner.
 
 Below you can see an example of intellisense within Visual Studio when we create a new RSVP object and add it to a Dinner's RSVPs collection. Notice how LINQ to SQL automatically added a "RSVPs" collection on the Dinner object:
 
-![](build-a-model-with-business-rule-validations/_static/image7.png)
+![Screenshot of the intellisense within Visual Studio. R S V Ps is highlighted.](build-a-model-with-business-rule-validations/_static/image7.png)
```

Comment:
R S V Ps or R S V Ps collection?

Reply:
R S V Ps is what is shown in the image. :)